src/commands/status/index.js

```diff
@@ -10,6 +10,22 @@ class StatusCommand extends Command {
   async run() {
     const { globalConfig, api, site } = this.netlify
     const { flags } = this.parse(StatusCommand)
+
+    if (flags.verbose) {
+      this.log()
+      this.log(`────────────────────┐
+ Environment Info   │
+────────────────────┘`)
+      const data = await envinfo.run({
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        npmGlobalPackages: ['netlify-cli']
+      })
+      this.log(data)
+      this.exit()
```

Comment:
Why exit here?

Reply:
Yeah, It would make sense to log rest of the status information.